src/main/java/racinggame/domain/Racing.java

```diff
@@ -0,0 +1,64 @@
+package racinggame.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Racing {
+    private static final int MAX_RANGE = 10;
+    private static final int FORWARD_NUMBER = 4;
+
+    private List<Car> cars = new ArrayList<>();
+    private StringBuilder result = new StringBuilder();
+
+    public Racing() {
+    }
+
+    public void race(List<String> carNames, int tryCount) {
+        registerCar(carNames);
+        while (tryCount > 0) {
+            move();
+            calculateResult();
+            tryCount--;
+        }
+    }
+
+    private void registerCar(List<String> carNames) {
+        for (String carName : carNames) {
+            cars.add(Car.of(carName));
+        }
+    }
+
+    private void move() {
+        for (Car car : cars) {
+            int randomNumber = makeRandomNumber();
+            if (randomNumber >= FORWARD_NUMBER) {
+                car.moveForward();
+            }
+        }
+    }
+
+    private int makeRandomNumber() {
+        Random random = new Random();
+        return random.nextInt(MAX_RANGE);
+    }
+
+    private void calculateResult() {
+        for (Car car : cars) {
+            result.append(car.getName() + " : ");
```

Comment:
String + String보다는 `append`로 한번 더 쪼개면 좋을 것 같아요!

```suggestion
            result.append(car.getName()).append(" : ");
```